pctasks_funcs/StorageEventsCF/__init__.py

```diff
@@ -0,0 +1,118 @@
+"""
+Cosmos DB Change Feed Dispatcher
+
+This Azure Function dispatches updates to the `storage-events` container in
+CosmosDB to the appropriate Blob Storage Queue. It's expected that PCTasks
+workflows will scale in response to messages being sent to that queue.
+"""
+import contextlib
+import logging
+import os
+import re
+from typing import Optional
+
+import azure.functions as func
+import azure.identity.aio
+import azure.storage.queue.aio
+
+from pctasks.core.models.event import StorageEvent
+
+# TODO: pre-commit-style validator for ensuring these queues exist.
+# TODO: Move to prefix / suffix rules?
+QUEUE_ENDPOINT = "https://pclowlatency.queue.core.windows.net/"
+
+RULES = [
```

Comment:
How do we feel about these dispatch rules being in code, rather than configuration? I have a sketch for how we might do it via configuration, but wanted to check what feels better. I'm really not sure.

Reply:
I think configuration that could be stored in e.g. Azure Tables would be best

Reply:
Initially, I was worried about the overhead of an extra HTTP request to look up the configuration on every invocation of the function. But we'll be already have at least 3 HTTP requests (one to get the message, a second to forward to the destination queue, and a third to delete the original message).

And if we have multiple documents in the `DocumentList` then we'll amortize the config lookup over all of them. Probably not worrying about, but I'll keep an eye on the performance metrics.

Reply:
I think I'll punt on how we do this queue routing till after merging this PR. But generally speaking, moving to some kind of configuration system makes sense.